doc/tutorial/argument_parser/index.md

```diff
@@ -310,10 +310,10 @@ Summary:
 
 | Tag        | Description                                                    |
 |:-----------|:---------------------------------------------------------------|
-| DEFAULT    | The default tag with no special behaviour.                     |
-| REQUIRED   | Required options will cause an error if not provided.          |
-| ADVANCED   | Advanced options are only displayed wit `-hh/--advanced-help`. |
-| HIDDEN     | Hidden options are never displayed when exported.              |
+| defaulted  | The default tag with no special behaviour.                     |
```

Comment:
I think the meaning of defaulted as adjective is wrong here. So `default` was actually correct.
And I just realise that we use this misleading word for all the defaulted constructors 🤯.

Reply:
I **cannot** use default :( ! (in lower case letters) because it is a compiler keyword. So I thought defaulted is the next best alternative. If you deem defaulted wrong, how else should we name it?

Reply:
But the meaning is rather: `required option`, `hidden option` etc. As far as I remember, we use a verb for functions only (it would be good if I could point now to the style guide but we don't have it). Nonetheless, I never've seen an enum value as a verb so it would be confusing to start now. Another, question, why do we have a default tag? Would that not be something internal, if the user just does not specify anything? What is the default behaviour? Why do you need to say explicitly it is default, which is already the meaning if I don't specify the other? Maybe we can remove it?

Reply:
> Nonetheless, I never've seen an enum value as a verb so it would be confusing to start now.

hm.. I see. I have no strong feelings about this. 

> Another, question, why do we have a default tag?

Because you need to specify the option_spec if you want to specify a validator.

```
p.add_option(var, 'i', "integer", "desc", option_spec::default, arithmetic_validator{3, 5});
```

> What is the default behaviour?

Neither, required, advanced nor hidden. Just a  normal option.

Possible alternatives (meaning "nothing special"): `standard`, `normal`, `simple`, `orderly`, `regular`, `classic`, `usual`.

Reply:
The default is public, because you need to provide it if you use a validator:

```cpp
    parser.add_option(args.sam_path, 'o', "output", "The output SAM file path.",
                      seqan3::option_spec::DEFAULT,
                      seqan3::output_file_validator{seqan3::output_file_open_options::create_new, {"sam"}});
```

Maybe we could use something like `standard` (which is a synonym of default) or even `normal`

Reply:
New idea: option_spec has "require" semantics.

The enum names of `seqan3::option_spec` were changed to lower case:
  * `seqan3::option_spec::DEFAULT` is replaced by `seqan3::option_spec::optional` (default-visible and non-required).
  * `seqan3::option_spec::REQUIRED` is replaced by `seqan3::option_spec::required` (default-visible and required).
  * `seqan3::option_spec::ADVANCED` is replaced by `seqan3::option_spec::advanced` (visible and non-required).
  * `seqan3::option_spec::HIDDEN` is replaced by `seqan3::option_spec::hidden` (non-visible and non-required).